ArduPlane/Parameters.h

```diff
@@ -93,14 +93,14 @@ class Parameters {
         k_param_scheduler,
         k_param_relay,
         k_param_takeoff_throttle_delay,
-        k_param_skip_gyro_cal, // unused
+        k_param_takeoff_rudder_mix,
```

Comment:
@WickedShell Do you mean the that those indexes are held to be used in transition table for transferring the value to newly set parameter space as done by this code: https://github.com/ArduPilot/ardupilot/blob/master/ArduPlane/Parameters.cpp#L1252-L1306 . I believe any index that's not present there is not being used for that purpose either, hence if say someone is to directly jump from a version where that parameter was used the newer parameter version won't be updated. 
One major issue might be that the index I swapped with might end-up containing bad value.